docs-2.0/5.configurations-and-logs/1.configurations/4.storage-config.md

```diff
@@ -182,14 +182,13 @@ rocksdb options 配置的格式为`{"<option_name>":"<option_value>"}`，多个
 | :-----------------------------| :------| :------------------------------- |
 | `enable_storage_cache`        | `false`|  是否开启 Storage 缓存功能。|
 | `storage_cache_capacity`      | `0`    |  为 Storage 缓存预留的内存大小，取值需要稍大于`vertex_pool_capacity`和`empty_key_pool_capacity`的总和。单位：兆字节。|
-| `storage_cache_buckets_power` | `20`   |  bucket 的个数。取值为以 2 为底的对数，取值范围：0~32。例如取值为 20，表示 bucket 个数为 2$^{20}$。建议值为`ceil(log2(cacheEntries * 1.6))`。`cacheEntries`为要缓存的条目总数。|
-| `storage_cache_locks_power`   | `10`    |  锁的个数。取值为以 2 为底的对数，取值范围：0~32。例如取值为 10，表示锁个数为 2$^{10}$。建议值为`max(1, storage_cache_buckets_power - 10)`。|
+| `storage_cache_entries_power` | `20`   |  预估 Storage 节点上缓存条目的个数，计算公式：2$^{<设置值>}$。例如当设置的值为`20`时，表示缓存条目的个数为 2$^{20}$。建议设置的值为`log2(该 Storage 中点的数量 * Tag 的数量)`，最大值可设置为`31`。|
```

Comment:
My bad. Since we have changed the underlying cache implementation. The estimated cache entries power is not log2(该 Storage 中点的数量 * Tag 的数量) any more. It can be log2(该 Storage 中点的数量).